README.md

```diff
@@ -1079,8 +1079,12 @@ Flags:
   - :nerd_face: `--format=wide`: Alias of `--format=table`
   - :nerd_face: `--format=json`: Alias of `--format='{{json .}}'`
 - :nerd_face: `--size`: Display the disk usage of volumes.
-
-Unimplemented `docker volume ls` flags: `--filter`
+- :whale: `-f, --filter`: Filter volumes based on given conditions.
+    - :whale: `--filter label=<key>=<value>`: Matches volumes by label on both
+      `key` and `value`. If `value` is left empty, matches all volumes with `key`
+      regardless of its value
+    - :whale: `--filter name=<value>`: Matches all volumes with a name containing
+      the `value` string
```

Comment:
Could you document unimplemented filter such as `dangling`?
https://docs.docker.com/engine/reference/commandline/volume_ls/

See also the document of `nerdctl images --filter`